aws_advanced_python_wrapper/resources/aws_advanced_python_wrapper_messages.properties

```diff
@@ -40,6 +40,42 @@ AwsSecretsManagerPlugin.UnhandledException=[AwsSecretsManagerPlugin] Unhandled e
 AwsWrapperConnection.ConnectionNotOpen=[AwsWrapperConnection] Attempted to establish an initial connection, but the connection returned by the connect call evaluated to None.
 AwsWrapperConnection.InitialHostInfoNone=[AwsWrapperConnection] The initial connection host info unexpectedly evaluated to None after initializing the host list provider.
 
+BlueGreenPhase.UnknownStatus=[BlueGreenPhase] Unknown blue/green status '{}'.
+
+BlueGreenRole.UnknownRole=[BlueGreenRole] Unknown blue/green role '{}'.
+BlueGreenRole.UnknownVersion=[BlueGreenRole] Unknown blue/green version '{}'.
+
+BlueGreenStatusMonitor.CreateHostListProvider=[BlueGreenStatusMonitor] [{}] Creating a new HostListProvider, clusterId: {}.
+BlueGreenStatusMonitor.Exception=[BlueGreenStatusMonitor] [{}] currentPhase: {}, exception while querying for blue green status.
+BlueGreenStatusMonitor.HostInfoNone=[BlueGreenStatusMonitor] Unable to initialize HostListProvider since connection host information is None.
+BlueGreenStatusMonitor.Interrupted=[BlueGreenStatusMonitor] [{}] Interrupted.
+BlueGreenStatusMonitor.MonitoringUnhandledException=[BlueGreenStatusMonitor] [{}] Unhandled exception while monitoring blue/green status: {}.
+BlueGreenStatusMonitor.NoEntriesInStatusTable=[BlueGreenStatusMonitor] [{}] No entries in status table.
+BlueGreenStatusMonitor.OpenedConnection=[BlueGreenStatusMonitor] [{}] Opened monitoring connection to {}.
+BlueGreenStatusMonitor.OpenedConnectionWithIp=[BlueGreenStatusMonitor] [{}] Opened monitoring connection (IP) to {}.
+BlueGreenStatusMonitor.OpeningConnection=[BlueGreenStatusMonitor] [{}] Opening monitoring connection to {}.
+BlueGreenStatusMonitor.OpeningConnectionWithIp=[BlueGreenStatusMonitor] [{}] Opening monitoring connection (IP) to {}.
+BlueGreenStatusMonitor.StatusChanged=[BlueGreenStatusMonitor] [{}] Status changed to: {}
+BlueGreenStatusMonitor.StatusNotAvailable=[BlueGreenStatusMonitor] [{}] (status not available) currentPhase: {}
+BlueGreenStatusMonitor.ThreadCompleted=[BlueGreenStatusMonitor] [{}] Blue/green status monitoring thread is completed.
+BlueGreenStatusMonitor.UnexpectedDialect=[BlueGreenStatusMonitor] Attempted to create a BlueGreenStatusMonitor, but a BlueGreenDialect is required. The current dialect is {}.
+BlueGreenStatusMonitor.UnhandledException=[BlueGreenStatusMonitor] [{}] Unhandled exception: {}.
+BlueGreenStatusMonitor.UsesVersion=[BlueGreenStatusMonitor] [{}] Blue/Green deployment uses version '{}' which the driver doesn't support. Version '{}' will be used instead.
+
+BlueGreenStatusProvider.BlueDnsCompleted=[BlueGreenStatusProvider] [bgdId: '{}'] Blue DNS update completed.
+BlueGreenStatusProvider.GreenDnsRemoved=[BlueGreenStatusProvider] [bgdId: '{}'] Green DNS removed.
+BlueGreenStatusProvider.GreenNodeChangedName=[BlueGreenStatusProvider] Green node '{}' has changed its name to '{}'.
```

Comment:
For consistency could you please replace instances of "node" with "host" in this wrapper?